src/components/tasks/card/index.tsx

```diff
@@ -67,21 +87,57 @@ const Card: FC<Props> = ({
   }
 
   function handleChange(event: any, changedProperty: keyof typeof cardDetails) {
+    const toChange: any = cardDetails;
```

Comment:
Don't we have interface of cardDetails? which could be assigned to toChange?

Reply:
we have `task` type, in that the types of `task` object is not consistent with backend validation, maybe a new PR needs to be raised to fix